WebRoot/js/zimbraMail/prefs/view/ZmZimletsPage.js

```diff
@@ -555,11 +555,17 @@ ZmPrefZimletListView.prototype._getCellId = function(item, field, params) {
 	return DwtId.getListViewItemId(DwtId.WIDGET_ITEM_CELL, "zimlets", item.name, field);
 };
 
+ZmPrefZimletListView.prototype._getCellClass = function (item, field) {
+	if (field === ZmPrefZimletListView.COL_ACTIVE) {
+		return "ZmAlignCenter";
+	}
+};
+
 ZmPrefZimletListView.prototype._getHeaderList =
 function() {
 	var hlist = [
-		(new DwtListHeaderItem({field:ZmPrefZimletListView.COL_ACTIVE, text:ZmMsg.active, width:ZmMsg.COLUMN_WIDTH_ACTIVE})),
-        (new DwtListHeaderItem({field:ZmPrefZimletListView.COL_NAME, text:ZmMsg.name, width:ZmMsg.COLUMN_WIDTH_FOLDER_DLV})),    
+		(new DwtListHeaderItem({field:ZmPrefZimletListView.COL_ACTIVE, text:ZmMsg.active, width:ZmMsg.COLUMN_WIDTH_ZIMLET_ACTIVE})),
+        (new DwtListHeaderItem({field:ZmPrefZimletListView.COL_NAME, text:ZmMsg.name, width:ZmMsg.COLUMN_WIDTH_ZIMLET_NAME})),
```

Comment:
can we correct the indentation here? I see that it wasn't right before as well but its better if we can correct that now.